examples/simple-file-server/README.md

```diff
@@ -0,0 +1,74 @@
+# Simple File Server
+
+## Overview
+
+This guide explains how to put and get a file to/from a UDP server using Ballerina.
+The following figure illustrates a high-level design diagram of the complete use case.
+
+![Sending a File with UDP](./simple-udp-server.jpg)
+
+The below are the detailed explanations of each of the steps of this example.
+
+### Step 1 - Start the UDP Server
+
+Execute the run command of the Ballerina Server project of the sample UDP server. This would start the UDP server.
+
+### Step 2 - Initialize the UDP Client with Credentials
+
+In order to interact with a UDP server, a UDP Client has to be initialized with the server related connection details and user's credentials. The Ballerina UDP client is configured to run on port the `20211` of `localhost`.
+
+### Step 3 - Send the File
+
+A blocked stream of bytes of the local file is created and passed to the UDP client along with the destination path name.
+Then the file is stored in the UDP file server.
+
+### Step 4 - Close the File Stream
+
+When all the file related operations are finished, the byte stream corresponding to the received file is closed.
+
+## Implementation
```

Comment:
Let's move it to the top. We can have it just below the `Overview` section.

Reply:
Move to that location.